tests/unit/api/auth.py

```diff
@@ -95,68 +95,68 @@ def server_init():
 auth_call_params = [
     # network_create
 
-    ### Legal cases:
```

Comment:
Ick, the way I wrote this these were actually supposed to be heirarchical headings.  Can you find a way to get it to not complain without flattening the tree? Does it accept `# ##`?